### DISCUSSION TITLE
Horizontal buttons stack

### DISCUSSION THREAD

**Discussion by fin05** [Clooos/Bubble-Card#494] [Questions about config, custom styles and templates] [answered]
How can I stack the buttons into 2 rows instead of scrolling to see all the buttons?

**Comment by Philipameen** [comment]
I would also like to know!

**Comment by DanteWinters** [comment]
Bumping with a comment for this feature request. I'd like to have a button that can be placed anywhere that brings up the hidden page.

**Comment by Clooos** [comment, marked as answer]
For now you can still create a regular button with a tap action that is set to "Navigate" with your pop-up hash.
